engine/login.php

```diff
@@ -108,8 +108,8 @@
 					<!--[if IE ]> <input type="hidden" name="auth_browser" value="invalid" /> <![endif]-->
 					<!--[if !(IE)]><!--> <input type="hidden" name="auth_browser" value="supported" /> <!--<![endif]-->
                     <?php if ($options['HOSTING_PROFILE']) { ?>
-                        <a href="<?php echo $login_action . '?provider=facebook&amp;remote_redirect=' . urlencode($options['SITE_HOST_ADDRESS'] . $ENGINE_ABS_ROOT . 'login.php') ?>" class="social_button social_button_facebook"><span class="icon-facebook"></span>Log in with Facebook</a>
-                        <a href="<?php echo $login_action . '?provider=google&amp;remote_redirect=' . urlencode($options['SITE_HOST_ADDRESS'] . $ENGINE_ABS_ROOT . 'login.php')  ?>" class="social_button social_button_google"><span class="icon-google-plus"></span>Log in with Google</a>
+                        <a href="<?php echo $login_action . '?provider=facebook&amp;remote_redirect=' . urlencode($options['SITE_HOST_ADDRESS'] . $ENGINE_ABS_ROOT . '_api/auth/login') ?>" class="social_button social_button_facebook"><span class="icon-facebook"></span>Log in with Facebook</a>
```

Comment:
are these URLs correct? wouldn't it result in `/eninge/_api/auth/login`?

Reply:
This should be encoded it separates `remote_redirect` and `provider` params.